ZenPacks/zenoss/WBEM/datasources/WBEMDataSource.py

```diff
@@ -30,11 +31,31 @@
 from ZenPacks.zenoss.PythonCollector.datasources.PythonDataSource \
     import PythonDataSource, PythonDataSourcePlugin
 
-from ZenPacks.zenoss.WBEM.utils import addLocalLibPath, result_errmsg
+from ZenPacks.zenoss.WBEM.modeler.wbem import check_if_complete
+
+
+from ZenPacks.zenoss.WBEM.utils import (
+    addLocalLibPath,
+    result_errmsg,
+    create_connection,
+)
 
 addLocalLibPath()
 
-from pywbem.twisted_client import ExecQuery
+from pywbem.twisted_client import (
+    ExecQuery,
+    OpenEnumerateInstances,
+)
+
+re_compile = re.compile(r'from+ ([\w\_]+)', re.I)
```

Comment:
It's better to use `\s` instead of whitespaces, to avoid inconsistencies in the future, for instance, there will be an issue if you change the regex mode to verbose.